packages/sysutils/eventlircd/evmap/aftvsremote.evmap

```diff
@@ -0,0 +1,11 @@
+KEY_KPENTER      = KEY_OK
+KEY_UP           = KEY_UP
+KEY_LEFT         = KEY_LEFT
+KEY_RIGHT        = KEY_RIGHT
+KEY_DOWN         = KEY_DOWN
+KEY_MENU         = KEY_TITLE
```

Comment:
@lrusak I think this is causing the menu button on the Fire TV remote to only work as the next subtitle button `<subtitle>NextSubtitle</subtitle>` instead of something more useful like the context menu `<menu>ContextMenu</menu>`